### PR TITLE
Enhance deploy.sh to support local storage

### PR DESCRIPTION
**Pull request by jinchihe**
fixes: #726

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/755)

### CODE REVIEW

**Comment by jinchihe** [comment]
/assign @IronPan

To more clear, I closed the old PR #727 which is related the issue #726 . Please review this one. You can check the previous reviewing process in #727 if needed. Thanks a lot.

File: components/kubeflow/deployer/src/deploy.sh [hunk 1]
```diff
@@ -95,6 +100,14 @@ echo "Generating the TF Serving config..."
 ks generate tf-serving server --name="${SERVER_NAME}"
 ks param set server modelPath "${MODEL_PATH}/export/export"
 
+# support local storage to deploy tf-serving.
+if [ ! -z "${PVC_NAME}" ];then
```
Comment (gaoning777) [inline components/kubeflow/deployer/src/deploy.sh]: Can it be [ -n "${PVC_NAME}" ] since this is more readable

**Comment by gaoning777** [comment]
Thanks for your contribution @jinchihe and sorry for the PR review delay.

File: components/kubeflow/deployer/src/deploy.sh (hunk 1 above)
Comment (jinchihe) [reply components/kubeflow/deployer/src/deploy.sh]: @gaoning777 Thanks for your comments. Updated.

**Comment by jinchihe** [comment]
> Thanks for your contribution @jinchihe and sorry for the PR review delay.

Never mind, your comment has been addressed. Please review. Thanks.

**Pull request merged**